catalog/nsili/catalog-nsili-transformer/src/main/java/com/connexta/alliance/nsili/transformer/DAGConverter.java

```diff
@@ -1140,23 +1137,26 @@ private static String collectionToString(Collection<String> collection) {
 
     public static void logMetacard(Metacard metacard, String id) {
         MetacardType metacardType = metacard.getMetacardType();
-        LOGGER.debug("{} : ID : " + metacard.getId(), id);
-        LOGGER.debug("{} :  Metacard Type : " + metacardType.getClass()
+        LOGGER.trace("{} : ID : " + metacard.getId(), id);
+        LOGGER.trace("{} :  Metacard Type : " + metacardType.getClass()
                 .getCanonicalName(), id);
-        LOGGER.debug("{} :  Title : " + metacard.getTitle(), id);
+        LOGGER.trace("{} :  Title : " + metacard.getTitle(), id);
         if (metacard instanceof MetacardImpl) {
             LOGGER.debug("{} :  Description : " + ((MetacardImpl) metacard).getDescription(), id);
         }
-        LOGGER.debug("{} :  Content Type Name : " + metacard.getContentTypeName(), id);
-        LOGGER.debug("{} :  Content Type Version : " + metacard.getContentTypeVersion(), id);
-        LOGGER.debug("{} :  Created Date : " + metacard.getCreatedDate(), id);
-        LOGGER.debug("{} :  Effective Date : " + metacard.getEffectiveDate(), id);
-        LOGGER.debug("{} :  Location : " + metacard.getLocation(), id);
-        LOGGER.debug("{} :  SourceID : " + metacard.getSourceId(), id);
-        LOGGER.debug("{} :  Modified Date : " + metacard.getModifiedDate(), id);
-        if (metacard.getResourceURI() != null) {
-            LOGGER.debug("{} :  Resource URI : " + metacard.getResourceURI()
-                    .toString(), id);
+        LOGGER.trace("{} :  Content Type Name : " + metacard.getContentTypeName(), id);
```

Comment:
Changed to trace to be consistent. Figured that would be best so we could go to debug without getting completely overwhelmed with data from all metacards being tranformed.